### ISSUE TITLE
[Bug] Loading state bar keeps going after everything is loaded.

### ISSUE THREAD

**Issue by pxpm** [Laravel-Backpack/theme-tabler#23]
I noticed this on the login page. 

Even after the page finish loading, the `loading bar` on top keeps going forever. 

![image](https://user-images.githubusercontent.com/7188159/234546957-27cb2f9e-c4e1-4b32-a1ed-6b30b277b630.png)

**Comment by tabacitu** [comment]
Yup, I also noticed that and was waiting for the polish stage to say something - we do need to fix that.

**Comment by maurohmartinez** [comment]
Is this related to basset? or is it related to Tabler theme?
